components/mpas-ocean/src/shared/mpas_ocn_constants.F

```diff
@@ -117,6 +118,7 @@ subroutine ocn_constants_init(configPool, packagePool)!{{{
        !-----------------------------------------------------------------------
 
        T0_Kelvin = 273.16_RKIND               ! zero point for Celsius
+       pi        = 3.141592_RKIND             ! pi
```

Comment:
This would be the value used in standalone (init and forward) mode so I think it needs to still be the same value as `SHR_CONST_PI` in that mode.

```suggestion
       pi        = 3.14159265358979323846_RKIND ! pi
```

Reply:
And also make the framework's `pii` match this value? `pii` is missing the last 6 decimal digits

Reply:
I think we just don't want to use the framework's `pii`.

Reply:
Changing that would need to be coordinated with MPAS-Seaice and MALI.

Reply:
It looks like both MPAS-Seaice and MALI attempt to override the framework's pi in their constants submodule. I looked through each of the components files and only found one instance where the framework's pi was used https://github.com/E3SM-Project/E3SM/blob/41bd0cbe6238e2e5aefb06338faf93f965f303ce/components/mpas-albany-landice/src/mode_forward/mpas_li_advection_fct_shared.F#L468

Given that we have a bit of time, do you think it would be appropriate to broaden the scope of this PR or just ask that the developers of each check this?